packages/twenty-front/src/modules/action-menu/actions/record-actions/utils/inheritActionsFromDefaultConfig.ts

```diff
@@ -0,0 +1,27 @@
+import { DEFAULT_RECORD_ACTIONS_CONFIG } from '@/action-menu/actions/record-actions/constants/DefaultRecordActionsConfig';
+import { DefaultRecordActionConfigKeys } from '@/action-menu/actions/types/DefaultRecordActionConfigKeys';
+import { RecordConfigAction } from '@/action-menu/actions/types/RecordConfigAction';
+
+export const inheritActionsFromDefaultConfig = (
```

Comment:
Could we name our parameters here in an object ? Because it's a bit hard to read :
- Not obvious that we have a call to `inheritActionsFromDefaultConfig` when reading a config object (we expect a static JS object, not a function call)
- Reading the big config object, we don't see that it's in fact multiple params stacked with a comma separating them.